### DISCUSSION TITLE
自定义翻译后的文字的颜色

### DISCUSSION THREAD

**Discussion by zwl1619** [immersive-translate/immersive-translate#480] [Ideas]
### 新功能 | New Feature

目前翻译后的文字都是黑色，能否增加自定义翻译后的文字的颜色的功能，这样可以让翻译结果更加显眼。

### 为什么您想要添加这些功能 | Why you want to add these features

可以让翻译结果更加显眼。

**Comment by theowenyoung** [comment]
界面设置-> 译文样式
